### PR TITLE
Update switch to TypeScript

### PR DESCRIPTION
**Pull request by berkbaski**

### CODE REVIEW

**Comment by berkbaski** [comment]
Could you check it again? @cenkce

File: ui/switch/switch.ios.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    this.nativeObject.layer.masksToBounds = false;
+
+    const onToggleChangedHandler = (toggle: boolean) => {
+      this._onToggleChanged(toggle);
+      this.emit(SwitchEvents.ToggleChanged, toggle);
+    };
+    this.nativeObject.addJSTarget(onToggleChangedHandler, UIControlEvents.valueChanged);
+
+    // Assign parameters given in constructor
+    for (const param in params) {
```
Comment (cenkce) [inline ui/switch/switch.ios.ts:34]: burayi kaldiralim

File: ui/switch/switch.ios.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import Color from '../color';
+import ViewIOS from '../view/view.ios';
+import { SwitchEvents } from './switch-events';
+
+export default class SwitchIOS<TEvent extends string = SwitchEvents> extends ViewIOS<TEvent | SwitchEvents, {}> implements ISwitch {
+  private _toggleOnColor: Color = Color.GREEN;
+  private _onToggleChanged: (toggle: boolean) => void;
+
+  constructor(params: Partial<ISwitch> = {}) {
+    super();
```
Comment (cenkce) [inline ui/switch/switch.ios.ts:13]: params gonderelim

File: ui/switch/switch.android.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        self._thumbOffColor = value;
+        self.setThumbColor();
+      }
+    };
+    const { android, ...restParams } = this;
+    Object.assign(this._android, _android);
+    Object.assign(this, restParams);
+
+    // Assign parameters given in constructor
+    for (const param in params) {
```
Comment (cenkce) [inline ui/switch/switch.android.ts:75]: burayi kaldiralim

**Review by cenkce — CHANGES_REQUESTED** [review]

**Comment by berkbaski** [comment]
I added `WithMobileOSProps` Could you recheck again? @cenkce

**Pull request merged**